recipes/squash/meta.yaml

```diff
@@ -0,0 +1,77 @@
+{% set name = "squash" %}
+{% set version = "0.7.0" %}
+{% set version_major = "0.7" %}
+{% set hash_type = "sha256" %}
+{% set hash_val = "4e741b1ab9dff9433fe9dd7393c7db41173b49b0426dcfcd03b13d0b03bc19b8" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.bz2
+  url: https://github.com/quixdb/squash/releases/download/v{{ version }}/{{ name }}-{{ version }}.tar.bz2
+  {{ hash_type }}: {{ hash_val }}
+  patches:
+    - patches/0001-Remove-wrongly-placed-break-on-cmake-file.patch
+    - patches/0002-Add-directive-to-have-_SC_PAGE_SIZE-defined.patch
+    - patches/0003-Fix-squash-missing-endian-declarations-in-OSX.patch
+    - patches/0004-Fix-zconf.h-generation.patch
+    - patches/0005-Fix-lzham-compilation-on-OSX.patch
+    - patches/0006-Fix-negative-left-shifting.patch
+    - patches/0007-Fix-negative-bit-left-shifting.patch
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - toolchain
+    - gcc  # [linux]
```

Comment:
So we supply `gcc` on Linux. Would recommend dropping this unless there is a special reason to keep it (e.g. needing Fortran or OpenMP).